manifests/server/installdb.pp

```diff
@@ -19,7 +19,7 @@
       creates   => "${datadir}/mysql",
       logoutput => on_failure,
       path      => '/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin',
-      require   => Package[$mysql::server::package_name],
+      require   => Package['mysql-server'],
```

Comment:
installdb itself is also wrapped in a `if $mysql::server::package_manage`. While this does make that var's name questionable, it does keep it from failing.

The service does not have that protection, and should have a similar construct as the `if $mysql::server::manage_config_file` at https://github.com/puppetlabs/puppetlabs-mysql/blob/master/manifests/server/service.pp#L39